permissions/plugin-vk-notifier.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "vk-notifier"
+github: "jenkinsci/vk-notifier-plugin"
+issues:
+- github: "jenkinsci/vk-notifier"
+paths:
+- "org/jenkins-ci/plugins/vk-notifier"
+developers:
+- "spliterash"
+cd:
+  enabled: true
```

Comment:
no one has implemented an action for gradle yet but no harm in enabling this here